src/coredump/coredumpctl.c

```diff
@@ -933,6 +995,11 @@ int main(int argc, char *argv[]) {
                 assert_not_reached("Shouldn't be here");
         }
 
+        if (units_active > 0)
+                printf("%s-- Warning: %d systemd-coredump@.service %s, output may be incomplete.%s\n",
```

Comment:
"Warning" appears a bit strong wording. I mean, warning are about something non-critical that is wrong, but there's nothing wrong about coredumps being processed, hence I am not sure "Warning" is the right wording. (also, I figure it makese sense to highlight this, but probably just with normal highlighting, and not necessarily with red highlighting)